### PR TITLE
Fixes attribute name of blocks_ids

### PR DESCRIPTION
**Pull request by jonasheinrich**
https://community.openproject.org/work_packages/21000

### CODE REVIEW

**Comment by TeatroIO** [comment]
I've prepared a stage to preview changes. <a href='http://jonasheinrich-fix-wrong-translation-in-backlogs.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com' target='_blank'>Open stage</a> or <a href='http://jonasheinrich-fix-wrong-translation-in-backlogs.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com/teatro.log' target='_blank'>view logs</a>.

**Comment by NobodysNightmare** [comment]
Good to know that this attribute is defined in the core and not in backlogs (because that is the only place it is being used) ^^

File: config/locales/en.yml [hunk 1]
```diff
@@ -307,7 +307,7 @@ en:
     attachments: "Attachments"
     author: "Author"
     base: "General Error:"
-    blocks_ids: "Blocks"
+    blocks_ids: "IDs of blocked work packages"
```
Comment (NobodysNightmare) [inline config/locales/en.yml:310]: Just for reference: `openproject-backlogs` defines a localization string called `label_blocks_ids` which translates to

> Blocks (IDs)

You might want to harmonize that in one direction or another.

Comment (jonasheinrich) [inline config/locales/en.yml:310]: :+1: I've done that!

**Comment by NobodysNightmare** [comment]
I am not sure how consistent this is with the rest of our application. Shouldn't we simply call it "blocked work packages"?

Things that make me think that:
- The presentation of that attribute might not always accept an ID, but would ideally accept a "work package" and the user does not notice how it is passed
- even when we let the user enter an ID, this is usually not our label (see the legacy WP edit form, when you enter a parent you have to pass an ID, but the label reads "Parent", not "Parent ID")

I see that in the context of the specific error message mentioned [here](https://community.openproject.org/work_packages/21015), it makes sense to include the "ID" term. 

But I'd rather fix the error message itself here (`can_only_contain_work_packages_of_current_sprint` in backlogs):

> can only contain ~~the IDs of~~ current sprint's tickets

**Comment by NobodysNightmare** [comment]
In any case you will break backlogs cukes in this file: `openproject-backlogs/features/scrum_master.feature`

``` cucumber
And the error alert should show "Blocks can only contain the IDs of current sprint's tickets"
```

**Comment by jonasheinrich** [comment]
Thanks for pointing out the cuke failure. I've only tested the specs. I'm going to fix that now.

Concerning the use of ID I'm not sure what would be the best. Since we have no autocompletion at this field I would recommend telling the user to enter IDs.

**Comment by NobodysNightmare** [comment]
Related PR: https://github.com/finnlabs/openproject-backlogs/pull/151

**Comment by jonasheinrich** [comment]
I fixed the cukes in the related pr. And I think we should merge this one as is, with the mentioning of IDs.

**Pull request merged**